region/brinstar/red/Red Tower.json

```diff
@@ -3147,8 +3242,22 @@
         "Alternatively, this can be done by shooting the block while falling right as it goes off screen,",
         "then using well-timed short space jumps to go up as rapidly as possible before the block respawns."
       ]
+    },
+    {
+      "id": 10,
+      "name": "Frozen Beetom X-Ray Climb with Morph",
```

Comment:
It looks very similar to the RMode beetom xray climb.  Are they different enough to be a new notable?

Reply:
I was thinking the R-mode version is more well known and could be a good candidate to move down to Expert at some point if/when the canRMode tech moves down. Whereas the version without R-mode is new and feels a bit more tricky, might be worth keeping in Extreme. They are related though, yeah